go/libkb/constants.go

```diff
@@ -712,6 +712,8 @@ const MinEphemeralKeyLifetime = MaxEphemeralContentLifetime + EphemeralKeyGenInt
 
 const MaxTeamMembersForPairwiseMAC = 100
 
+const TeamBackoffBeforeAuditOnNeedRotate = time.Minute
```

Comment:
even with jitter it feels like a lot of people will race at same time in a medium sized team, is that ok?